src/translations/po/it-IT.po

```diff
@@ -0,0 +1,174 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR This file is copyright:
+# This file is distributed under the same license as the plasma5-wallpapers-dynamic package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: plasma5-wallpapers-dynamic\n"
+"Report-Msgid-Bugs-To: https://github.com/zzag/plasma5-wallpapers-dynamic/"
+"issues\n"
+"POT-Creation-Date: 2019-10-25 00:53+0300\n"
+"PO-Revision-Date: 2019-10-26 15:00+0200\n"
+"Last-Translator: Federico Robin <bisio.97@gmail.com>\n"
+"Language-Team: Italian <LL@li.org>\n"
```

Comment:
`Language-Team` contains the mailing list on which the translation team can be reached. However, this project doesn't have such a team. Either delete this field or put your credentials as in `Last-Translator` field. It's up to you.